pkg/engine/api/ruleresponse.go

```diff
@@ -46,6 +47,9 @@ type RuleResponse struct {
 	exceptions []GenericException
 	// binding is the validatingadmissionpolicybinding (if any)
 	binding *admissionregistrationv1.ValidatingAdmissionPolicyBinding
+	// mutatingBinding is the mutatingadmissionpolicybinding (if any)
+	mutatingBinding *admissionregistrationv1alpha1.MutatingAdmissionPolicyBinding
+
```

Comment:
Can you remove that for now? It will be needed in case of generating reports for MAPs, but this PR is for the CLI.

Reply:
Can you remove that for now? It will be needed in case of generating reports for MAPs, but this PR is for the CLI.